cmd/gen-go-sample/testdata/sample_unary.want

```diff
@@ -36,6 +38,33 @@ func sampleUnaryMethod(theX int64, theB string, theFoo string) error {
 	// theX := 42
 	// theB := "foobar"
 	// theFoo := "myfoo"
+	// fileArg := "path/to/local/file/arg.txt"
+
+	var file io.Reader
+	var err error
+
+	file, err = os.Open("path/to/local/file/default.txt")
```

Comment:
This should presumably be `fileArg` instead of the literal string?

Reply:
There doesn't need to be any additional code to read the "default"

The default is merely a string file path. The code sample should do nothing other than `ReadFile` the bytes from the provided string parameter to the file path

Reply:
There are two file inputs here.